web_src/js/features/repo-projects.js

```diff
@@ -9,6 +9,20 @@ function updateIssueCount(cards) {
   parent.getElementsByClassName('board-card-cnt')[0].textContent = cnt;
 }
 
+function createNewBoard(boardTitle, projectColorInput) {
+  $.ajax({
+    url: $(this).data('url'),
```

Comment:
What does `this` point to? I guess it is wrong.

Reply:
Fixed